README.md

```diff
@@ -436,25 +366,19 @@ Now, on your own system run:
 cpu -key out/keys/cpu_keys/cpu_rsa <host>
 ```
 
-This will connect you to the remote server and bring all your tools and environment with it. Be aware that this process might take up to a few minutes depending on the size of your environment and the power of the remote machine.
+This will connect you to the remote server and bring all your tools and environment with it. Be aware that this process might take up to a few minutes, depending on the size of your environment and the power of the remote machine.
 
-You can test it by running in qemu:
+You can test it by running in QEMU:
 ``` bash 
-# run installation in qemu
+# run installation in QEMU
 task run
 # interrupt stboot while booting
 ctrl+c
 # start the cpu daemon
 ./elvish start_cpu.elv
 ```
 
-In the newly opened terminal run:
+In the newly opened terminal, run:
 ``` bash
 cpu -key keys/cpu_keys/cpu_rsa localhost
 ```
-
-# Development
-This repository uses its own path (`GOPATH=cache/go`) to build the needed Go binaries by default.
-To use the default system GOPATH, define `ST_DEVELOP=1` as an environment variable. It will prevent
-any `git checkout` operation, which could change the HEAD of any Go repository at build-time. Furthermore, you can
-define the environment variable `ST_GOPATH` to use a custom GOPATH for the installation.
```

Comment:
We do not use our own GOPATH anymore?

Reply:
We do, but this info is obsolete. `ST_DEVELOP`, and `ST_GOPATH` does not exist anymore.